templates/UmbracoPackage/.template.config/dotnetcli.host.json

```diff
@@ -2,12 +2,16 @@
   "$schema": "https://json.schemastore.org/dotnetcli.host.json",
   "symbolInfo": {
     "Framework": {
-      "longName": "Framework",
-      "shortName": "F"
+      "longName": "framework",
+      "isHidden": true
```

Comment:
The only allowed framework version is `net7.0`, so there's no need to show this parameter.

Since this value isn't supported on older Umbraco versions, any unaltered command would result in an `...is not a valid value...` error, so I've also updated the capitalization to align with the default ASP.NET Core templates.